configs/wf_configs/cp_process_singlecells.yaml

```diff
@@ -1,4 +1,36 @@
 name: cp_process_singlecells_configs
+
+# Documentation
+docs: |
+  Description:
+  ------------
+  Converts sqlite plate data into parquet and returns selected features in csv
+  format
+
+  Workflow Steps:
+  ---------------
+  Below the workflow steps are separated in chunks.
+
+  cytotable_convert:
+    Takes in sqlite file and converts it into a parquet file.
+
+    Uses CytoTable's convert workflow which can be found in:
+    https://github.com/cytomining/CytoTable/blob/main/cytotable/convert.py
```

Comment:
Consider referencing CytoTable's docsite API reference here: https://cytomining.github.io/CytoTable/python-api.html#cytotable.convert.convert . The difference is minor, but may increase the possibility for understanding.